android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -2244,6 +2306,34 @@ public static void applyKeyboardHeight(Context context, int height) {
     }
   }
 
+  public static int getDefaultKeyboardHeight(Context context) {
+    int orientation = getOrientation(context);
+    return getDefaultKeyboardHeight(context, orientation);
+  }
+
+  public static int getDefaultKeyboardHeight(Context context, int orientation) {
+    if (isTestMode()) {
+      // Keyboard height not needed for unit tests  #13578
+      return KeyboardHeight_Context_Portrait_Default;
```

Comment:
```suggestion
      return 0;
```

`KeyboardHeight_Context_Portrait_Default` will not have been set (this function is responsible for returning the value to be set, so it is less smelly to avoid a reference to it here, even though it will be zero.

Reply:
Actually, since we can't check default values here, shouldn't we pass a valid portrait height (maybe 300) here instead of 0?